app/controllers/user/attendances_controller.rb

```diff
@@ -1,5 +1,7 @@
 class User::AttendancesController < User::BaseController
-  
+  before_action :find_attendance_by_id, only: [:show, :edit, :update, :destroy]
```

Comment:
great refactor for multiple methods that were relying on find_attendance_by_attendance_id

Reply:
Thanks. Keepin it D.R.Y.